APE0.rst

```diff
@@ -0,0 +1,385 @@
+  organizers: core package developer telecons, co-working telecons,
+  infrastructure telecons, proposal-related telecons, and Astropy Learn
+  telecons.
+* Has commit rights to at least one repository in the astropy GitHub
+  organization which is either the core package, a coordinated package,
+  an infrastructure package, or an Astropy Learn-related repository,
+  and has actively used those commit rights in the last two years prior to the
+  acceptance of this APE.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can be nominated as a Voting Member by providing evidence of meeting the
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will put it to a vote among active Voting Members. Votes for adding
+  voting members will be carried out at most twice a year to avoid too many
+  elections. The voting dates will be announced with at least four weeks notice,
+  and additional nominations will be accepted until two weeks before the
+  election date. The voting will be open for at least two weeks. To successfully
+  gain an appointment as a Voting Member, the nominee must receive at least four
+  positive votes, as long as that constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+At least one round of voting for adding voting members should be carried out before
+the initial election of the Coordination Committee.
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as "emeritus.” Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the “emeritus” category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to emeritus. To record and honor their contributions,
+emeritus Voting Members will continue to be listed. emeritus Voting Members are
+not able to participate in votes.
```

Comment:
I italicized instead of scare-quoting the *emeritus* references.  At least, the quotes need to be made consistent.
```suggestion
Voting Members have no term or term limits. Voting Members who have stopped
contributing are encouraged to declare themselves as *emeritus*. Those who have
not made any significant contribution for two years may be asked to move
themselves to the *emeritus* category by the Coordination Committee. If no
response is received, the Coordination Committee may automatically change a
Voting Member’s status to *emeritus*. To record and honor their contributions,
*emeritus* Voting Members will continue to be listed. *Emeritus* Voting Members are
not able to participate in votes.
```